packages/preset-attributify/src/extractor.ts

```diff
@@ -37,7 +37,7 @@ export const extractorAttributify = (options?: AttributifyOptions): Extractor =>
           }
 
           if (!content) {
-            if (isValidSelector(name) && nonValuedAttribute !== false) {
+            if (isValidSelector(name) && nonValuedAttribute !== false && !name.endsWith(':')) {
```

Comment:
Would this be too specific? Should we improve the rules to not handle those cases instead?

Reply:
Done 👍